client/src/components/App.jsx

```diff
@@ -4,21 +4,39 @@ import Atelier from '../Atelier.js';
 // import RatingsReview from './RatingsReview.jsx';
 import Overview from './overview/Overview.jsx';
 import Ratings from './RatingsReview.jsx';
+import RICWidget from './RIC-Widget.jsx';
 
 class App extends React.Component {
   constructor(props) {
     super(props);
     this.state = {
-      holder: ''
+      product: 13030,
```

Comment:
I assume we will eventually get these from the API, but this is fine for now.

Reply:
Correct! It's just for me to test my widget currently